.github/workflows/ci.yml

```diff
@@ -92,6 +92,7 @@ jobs:
     env:
       DATABASE_HOST: postgres
       AUTO_INCLUDED_KEYCHAIN_ID: 77777777-7777-7777-7777-777777777777
+      SLACK_API_TOKEN: not-real
```

Comment:
required so that i can assert that i sent myself a slack in the test case. without a slack token the slack client (which is mocked in tests) won't try to send and therefore i can't assert